js/script.js

```diff
@@ -0,0 +1,50 @@
+var searchResults = [];
+var slideshow; 
+
+$(document).ready(function() {
+	
+	$('#submit').on('click', function(e) {
+		e.preventDefault();
+		$('#form').hide();
+		$('#instructions').hide();
+		console.log($('input').val());
+		var searchinput = "https://www.reddit.com/search.json?q="+$('input').val()+"+nsfw:no";
+		console.log(searchinput);
+		$.get("https://www.reddit.com/search.json?q="+$('input').val()+"+nsfw:no").done(function(results) {
+			searchResults = results;
+			console.log(searchResults);
+			var children = searchResults.data.children;
+			console.log(children);
+// create array of image urls using filter and map
+			var length = function (object) {
+				if (object.data.url.length) {
```

Comment:
This works fine if the post has an image, but will return an invalid result if the post is a text post. `object.data.preview.images[0].source.url` or `object.data.thumbnail` would help guarantee that the links are images.